ddtrace/constants.py

```diff
@@ -22,6 +22,15 @@
 
 LOG_SPAN_KEY = "__datadog_log_span"
 
+ERROR_MSG = "error.msg"  # a string representing the error message
+ERROR_TYPE = "error.type"  # a string representing the type of the error
+ERROR_STACK = "error.stack"  # a human readable version of the stack. beta.
+
+# shorthand for -----^
+MSG = ERROR_MSG
```

Comment:
I'd like to get rid of these too.... Can we leave these in the `ext.errors` module and explicitly call it out in the release note?

Reply:
That works for me. There was only 2 references to shorthand version in our internal code and I replaced with with the corresponding ERROR_<> constant.